### DISCUSSION TITLE
So this works on opensuse - tumbleweed!

### DISCUSSION THREAD

**Discussion by SteveDevOps** [moraroy/NonSteamLaunchers-On-Steam-Deck#187] [General]
Just reporting in that this works on opensuse - tumbleweed - wayland. Since it's KDE.. it just works.  I was up and running with battle.net in no time. Appreciate all the work the team is doing!

**Comment by moraroy** [comment]
@SteveDevOps i have no idea what those are but im glad its working for you brother, just fyi i added Battle.net to the NSLGameScanner. It should get your games added!
